pkg/certificates/triple/cert/cert.go

```diff
@@ -51,11 +53,16 @@ type AltNames struct {
 	IPs      []net.IP
 }
 
-// NewPrivateKey creates an RSA private key
-func NewPrivateKey() (*rsa.PrivateKey, error) {
+// NewRSAPrivateKey creates an RSA private key
+func NewRSAPrivateKey() (*rsa.PrivateKey, error) {
 	return rsa.GenerateKey(cryptorand.Reader, rsaKeySize)
 }
 
+// NewECPrivateKey creates an ECDSA private key
+func NewECPrivateKey() (*ecdsa.PrivateKey, error) {
```

Comment:
Can you change this to `NewECDSAPrivateKey`? I don't think EC by itself says much.

Reply:
Done :+1: